src/test/java/com/tabnine/testUtils/HealthCheckTestUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.tabnine.testUtils;
+
+import com.intellij.openapi.application.ApplicationManager;
+import com.tabnine.binary.requests.config.CloudConnectionHealthStatus;
+import com.tabnine.binary.requests.config.StateResponse;
+import com.tabnine.lifecycle.BinaryStateChangeNotifier;
+
+public class HealthCheckTestUtils {
+
+  public static void notifyHealthStatusLoop(
```

Comment:
1. you can have this function to be exactly as the other one - `notifyHealthStatus` (function overloading)
2. why Java? we prefer new files to be written in Kotlin